### ISSUE TITLE
Dashes/hyphens replacement

### ISSUE THREAD

**Issue by allanaaa** [omeka/omeka-s-enduser#245]
Screenreaders aren't consistent on en or em dashes. Generally those grammatical structures should be replaced with commas and parenthetical inclusion, or colons where appropriate. 

More importantly we have used the basic hyphen symbol to indicate all of hypenated terms ("full-text"), en dash, em dash, a numerical range, and the mathematical minus symbol. 

The latter four uses should be found and eliminated. 

This work should start after https://github.com/omeka/omeka-s-enduser/tree/plain-language-edits is merged.
